api/domain/eventdata/certificate.proto

```diff
@@ -18,18 +18,21 @@ syntax = "proto3";
 // https://cloud.google.com/apis/design/naming_convention
 
 import "api/domain/common/messages.proto";
+import "validate/validate.proto";
 
 option go_package = "github.com/finleap-connect/monoskope/pkg/api/domain/eventdata";
 
 package eventdata;
 
 message CertificateRequested {
   // Unique identifier of the aggregate referenced (UUID 128-bit number)
-  string referenced_aggregate_id = 1;
+  string referenced_aggregate_id = 1 [(validate.rules).string.uuid = true];
   // Type of the aggregate referenced
-  string referenced_aggregate_type = 4;
+  string referenced_aggregate_type = 4 [(validate.rules).string = {pattern: "^[a-zA-Z_]+$", max_bytes: 60}];
```

Comment:
We need to allow numbers here for versioned aggregate types, e.g. "ClusterV2").

Reply:
not sure why GitHub showed you old code, but I already did that here 
https://github.com/finleap-connect/monoskope/pull/12/commits/0371ac14e4805158300b1578491101dcbc07edcf

starting with a number is not allowed right?

Reply:
right! 👍